regression/regression_param.sh

```diff
@@ -131,6 +144,9 @@ case $regtest in
         elif [[ "$machine" = "Cheyenne" ]]; then
```

Comment:
Just noting that when you get to Cheyenne, these wall clocks will likely need to be adjusted.

Reply:
Updated for cheyenne